tests/run_auth_tests.sh

```diff
@@ -15,4 +15,4 @@
     ./run_tests.sh shotgun_authentication_tests.test_session_cache &&
     ./run_tests.sh util_tests.test_login &&
     ./run_tests.sh util_tests.test_shotgun &&
-    ./run_tests.sh shotgun_authentication_tests.test_interactive_authentication --interactive
+    ./run_tests.sh shotgun_authentication_tests.test_interactive_authentication $*
```

Comment:
I was getting annoyed at having to sit through the interactive tests all the time when I didn't need to run them, so instead they are now optional.